### ISSUE TITLE
Edit function does not update contents of the rdf.yaml

### ISSUE THREAD

**Issue by constantinpape** [bioimage-io/archive.bioimage.io#246]
When editing an entry, removing the rdf.yaml and uploading a new one, the contents of the new rdf.yaml are not used (but rather the content of the existing one is still used). See for example the video below, where I update the rdf for a model with `min_shape: [64, 64]`, but it still uses the old `min_shape: [32, 32]` (and then fails validation because this min_shape is not large enough for the given halo):
https://user-images.githubusercontent.com/4263537/156005328-4cfe941a-044f-42d7-8dba-56c0e1306eeb.mp4

**Comment by FynnBe** [comment]
I think uploading the RDF as attachment files is not the intended use of the uploader.
Adding file attachments is different than starting the upload process from a local rdf.yaml....

**Comment by constantinpape** [comment]
> I think uploading the RDF as attachment files is not the intended use of the uploader.
> Adding file attachments is different than starting the upload process from a local rdf.yaml....

But how do I update the rdf then? Also, if this is not the intended usage then the rdf should not be shown in the files.

**Comment by oeway** [comment]
Hi, have you tried the dedicated file selection button for RDF in the end of the form? We have discussed this issue before, the issues is that we have two sources for the RDF, the form and the files, and they may not agree with each other, the current implementation is to use the form to update the RDF file, if you want to replace the RDF file entirely in an update, you need to use the dedicated button to populate the form from an RDF file.

**Comment by constantinpape** [comment]
Yes, that works (well sort of... https://github.com/bioimage-io/spec-bioimage-io/issues/411).

Can we somehow make this option more obvious? E.g. by not enabling to replace the "rdf.yaml" in the files field?

**Comment by oeway** [comment]
> Can we somehow make this option more obvious? E.g. by not enabling to replace the "rdf.yaml" in the files field?

I agree that this need some improvements, the current limitation is that the UI element for the form does not allow us to do dynamic checks for the files,  however, we can perhaps check that if the user try to change the RDF file in files and complain to the user when they try to submit the form.
